docs/content/assets/css/customizations.css

```diff
@@ -8,7 +8,7 @@ If you want to append instead, switch `::before` to `::after`.
 /* reference the icon font to use */
 @font-face {
   font-family: 'external-link';
-  src: url('/docker-mailserver/assets/fonts/external-link.woff') format('woff');
+  src: url('../fonts/external-link.woff') format('woff');
```

Comment:
Does this fix the local dev environment too?

Reply:
Yup